cep-????.md

```diff
@@ -0,0 +1,171 @@
+
+### List of virtual packages
+
+In alphabetical order, every conda client MUST support the following virtual packages:
+
+- `__archspec`
+- `__cuda`
+- `__glibc`
+- `__linux`
+- `__osx`
+- `__unix`
+- `__win`
+
+#### `__archspec`
+
+This virtual package MUST be always present, with the version set to `1`. 
+
+The build string MUST reflect one of:
+
+- If the target platform matches the native platform, the best fitting CPU microarchitecture in the [`archspec/archspec-json` database](https://github.com/archspec/archspec-json/blob/v0.2.5/cpu/microarchitectures.json). The reference CPU detection implementation is [`archspec.cpu.detect.host()`](https://github.com/archspec/archspec/blob/v0.2.5/archspec/cpu/detect.py#L338).
+
+- The target platform architecture (second component of the platform string), mapped as: 
+
+  | Target platform | Reported `archspec` build string |
+  | --------------- | -------------------------------- |
+  | `*-32`          | `x86`                            |
+  | `*-64`          | `x86_64`                         |
+  | `*-armv6l`      | `armv6l`                         |
+  | `*-armv7l`      | `armv7l`                         |
+  | `*-aarch64`     | `aarch64`                        |
+  | `*-arm64`       | `arm64`                          |
+  | `*-ppc64`       | `ppc64`                          |
+  | `*-ppc64le`     | `ppc64le`                        |
+  | `*-riscv64`     | `riscv64`                        |
+  | `*-s390x`       | `s390x`                          |
+  | `zos-z`         | `0`                              |
+  | Any other value | `0`                              |
+
+The build string MUST be overridable with the `CONDA_OVERRIDE_ARCHSPEC` environment variable, if set to a non-empty value.
+
+#### `__cuda`
+
+This virtual package MUST be present when the system exhibits GPU drivers compatible with the CUDA runtimes. When available, the version value MUST be set to the oldest CUDA version supported by the detected drivers (i.e. the formatted value of `libcuda.cuDriverGetVersion()`), constrained to the first two components (major and minor) and formatted as `{major}.{minor}`. The build string MUST be `0`.
+
+The version MUST be overridable with the `CONDA_OVERRIDE_CUDA` environment variable, if set to a non-empty value that can be parsed as a version string.
+
+#### `__glibc`
+
+This virtual package MUST NOT be present if the target platform is not `linux-*`.
+
+This virtual package MUST be present when the native and target platforms are both the same type of `linux-*` and GNU `libc` is installed in the system. The version value MUST be set to the system GNU `libc` version, constrained to the first two components (major and minor) formatted as `{major}.{minor}`. If the version cannot be estimated, the tool MUST set the version to a default value (e.g. `2.17`). 
```

Comment:
Is it worth calling out an expected behavior on Linux systems where the `libc` is not GNU, for example those using [musl](https://musl.libc.org/).

Reply:
There has been a lot of discussion on this point already above: https://github.com/conda/ceps/pull/103#discussion_r1889312955

Essentially we have decided so far to...

* Focus on existing virtual packages like `__glibc` (not ones yet to be added)
* Standard system configurations (so no `glibc` compat layer)

There is a "Potential future work" section below that states `__musl` could be pursued in a future CEP

Reply:
This is covered by the callout below:
> More `libc` implementations, like `__musl`.